packages/frontend/src/components/AppBar/index.tsx

```diff
@@ -24,6 +30,10 @@ type AppBarProps = {
 
 const accountMenuId = 'account-menu'
 
+// this fetches the latest time from the news
+const latestNewsTimestamp =
+  TEST_ITEM_LIST.length > 0 ? TEST_ITEM_LIST[0].date.getTime().toString() : ''
+
```

Comment:
why put this chunk of code here? this belongs in NewsDrawer component

Reply:
thanks